splunk-otel-android/src/test/java/com/splunk/rum/StartTypeAwareSpanStorageTest.java

```diff
@@ -51,14 +51,15 @@ void setup() {
     @Test
     void getPendingFiles_givenInBackground_shouldReturnForegoundOnlySpan() {
         when(visibleScreenTracker.getPreviouslyVisibleScreen()).thenReturn(null);
+        when(visibleScreenTracker.getCurrentlyVisibleScreen()).thenReturn("unknown");
         List<File> spans = fileProvider.getPendingFiles().collect(Collectors.toList());
         assertEquals(0, spans.size());
     }
 
     @Test
-    void
-            getPendingFiles_givenPrevouslyInBackground_shouldMoveBackgroundSpanToForegroundSpanForSending() {
+    void getPendingFiles_givenPrevouslyInBackground_shouldMoveBackgroundSpanToForegroundSpanForSending() {
         when(visibleScreenTracker.getPreviouslyVisibleScreen()).thenReturn("MainActivity");
+        when(visibleScreenTracker.getCurrentlyVisibleScreen()).thenReturn("MainActivity");
```

Comment:
Might be a little weird that both the previously visible screen AND the currently visible screen have the same name. Can we make the previous one something different?